index.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const NR = require('node-resque');
+
+// SET UP THE CONNECTION
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD,
+    port: process.env.REDIS_PORT,
+    database: 0
+};
+
+// DEFINE YOUR WORKER TASKS
+const jobs = {
+    start: {
+        plugins: ['jobLock', 'retry'],
+        pluginOptions: {
+            jobLock: {},
+            retry: {
+                retryLimit: 3,
+                retryDelay: (1000 * 5)
+            }
+        },
+        /* config looks like this
+            executor:
+                name: k8s-vm
+                options:
+                    kubernetes: ...
+            buildConfig:
+                apiUri:
+                buildId:
+                container:
+                token:
+         */
+        perform: (config, callback) => {
+            // eslint-disable-next-line
+            const ExecutorPlugin = require(`screwdriver-executor-${config.executor.name}`);
```

Comment:
@petey 

> What happens if the executor plugin doesn't exist? Does this crash and take the entire worker down?

Based on [one of the examples](https://github.com/taskrabbit/node-resque/blob/master/examples/errorExample.js#L37-L39), it looks like it will. We should look into wrapping the entire `perform` function in a Promise to mitigate that scenario